### DISCUSSION TITLE
Move multiple dates on the roadmap

### DISCUSSION THREAD

**Discussion by timuric** [community/community#51482] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Body

I have a roadmap with 20+ items, if I need to move my timeline for week I have to individually move every single item, which requires a lot of effort specially when you want to keep exact sequence of tasks

**Comment by rileybroughten** [comment, marked as answer]
@timuric 👋  - I've added this feedback to the more general roadmap feedback discussion and will close this one out, feel free to add more comments on that thread and we'll post updates there! Thank you 👍

**Reply by msa0311** [reply]
In our eyes this is almost a blocker for seriously using the roadmap feature. Besides this its pretty good already! Is there a timeline for an improvement in this regards?

**Reply by rileybroughten** [reply]
An option that could help with this is to use the `Show date fields` option from the view configuration menu to make fast and bulk edits. This is a user setting that only applies to your view, but then you can utilize copy/paste and dragging cell values to update multiple at once.

Let me know if this helps!


https://github.com/community/community/assets/101840513/98fa7fa2-2ca4-4320-b500-b0b8baf08574

**Reply by timuric** [reply]
not really, imagine you crafted a rodamap without speints, with things following sequentially, now you need to move it by one week because of delay. It is truly unusable

**Reply by trusktr** [reply]
@rileybroughten this isn't really the answer. Can you please unselect it?

We need to be able to drag multiple items at once to change their _dates_.

**Comment by vizthex123** [comment]
shift+click to select multiple things would be good, and being able to remove stuff from the roadmap view would be nice as well.

**Comment by grantcurell** [comment]
Was this added? I'm working with a team and want to use github roadmap, but if you realize you need to insert one task and you have to manually move 50 things it's pretty much a no go.
